MainModule/Client/Core/Variables.lua

```diff
@@ -80,4 +81,4 @@ return function()
 			FogStart = service.Lighting.FogStart;
 		}
 	};
-end
\ No newline at end of file
+end
```

Comment:
Why did you remove the new line at the end of the file?
It's a standard programming convention.
```suggestion
end

```